cmd/incus/top.go

```diff
@@ -0,0 +1,282 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	incus "github.com/lxc/incus/v6/client"
+	cli "github.com/lxc/incus/v6/internal/cmd"
+	"github.com/lxc/incus/v6/internal/i18n"
+	"github.com/lxc/incus/v6/internal/server/metrics"
```

Comment:
What are you getting from this one?

The client should normally never import anything from `internal/server/XYZ`

Reply:
This brings in the MetricType struct and MetricNames enum that I'm using to work with the logs. I can also copy the file over here if we want - let me know if this is preferred!